 meson.build | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/meson.build b/meson.build
index 90b7dc1e3..b7dab745a 100644
--- a/meson.build
+++ b/meson.build
@@ -3056,15 +3056,15 @@ endif

 ############################################################

-meson_check_api_docs_sh = find_program('tools/meson-check-api-docs.sh')
-run_target(
-        'check-api-docs',
+#meson_check_api_docs_sh = find_program('tools/meson-check-api-docs.sh')
+#run_target(
+#        'check-api-docs',
 #if 0 /// libudev is external, elogind does not need to document it.
 #         depends : [man, libsystemd, libudev],
 #         command : [meson_check_api_docs_sh, libsystemd.full_path(), libudev.full_path()])
 #else
-        depends : [man, libelogind],
-        command : [meson_check_api_docs_sh, libelogind.full_path()])
+#        depends : [man, libelogind],
+#        command : [meson_check_api_docs_sh, libelogind.full_path()])
 #endif // 0

 ############################################################
